test/wdio/scoped-id-in-nested-classname/cmp.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('scope-id-in-nested-classname', function () {
+  beforeEach(() => {
+    render({
+      template: () => <cmp-level-1></cmp-level-1>,
+    });
+  });
+
+  it('should have root scope id in the nested element as classname', async () => {
+    await expect($('cmp-level-3')).toHaveElementClass('sc-cmp-level-1');
```

Comment:
Can we also assert the applied CSS directly via `getCSSProperty`? I think this would give us a bit more end to end confidence, wdyt?

Reply:
tbh, I can't see the value of adding this check since it's gonna be a test case-specific check, not the functionality. IMHO, checking the classname would be sufficient, checking one step further would be falling into browser/jsdom behavior testing area

Reply:
anyway, I added the test case